packages/connect-query/src/unary-hooks.test.ts

```diff
@@ -139,14 +139,11 @@ describe('unaryHooks', () => {
     const input = { sentence: 'ziltoid' } satisfies PartialMessage<SayRequest>;
     const transport = mockEliza();
 
-    const { result } = renderHook(
-      async () => {
-        const { queryFn } = genSay.useQuery(input);
+    const { result } = renderHook(async () => {
```

Comment:
We should not touch code in this PR, even if it is just formatting. Can you revert?